overlord/devicestate/devicemgr.go

```diff
@@ -670,7 +670,11 @@ func (m *DeviceManager) Ensure() error {
 	var errs []error
 
 	if err := m.ensureSeeded(); err != nil {
-		errs = append(errs, err)
+		// XXX: avoid duplicated warnings?
+		m.state.Lock()
+		m.state.Warnf("seeding failed, this indicates an error on your distribution, please contact your vendor: %v", err)
```

Comment:
s/on your/in your/ ?

"vendor" sounds slightly weird, given that some distros are community distros.   

just "contact them" ?

maybe @degville has suggestions

Reply:
I was wondering if we should include a url here to a forum topic, wdyt?

Reply:
(definitely `in your`) and I agree about _vendor_ sounding slightly weird - it would be good if we could offer something more constructive. Is there a vendor/distribution string/URL we can output?  Failing that, maybe 'device maintainer'. I think the forum topic is a good idea too - better than a dead end.

Reply:
@mvo5 suggestion on the warning, could you warn about the md5 of the seed? I think, at least, we would get an idea about which seed is broken this way.

Reply:
I don't think we will need the md5 in the end, we will disable the seed in a followup automatically if it's broken.